lib/core/Tile.simba

```diff
@@ -87,15 +87,16 @@ begin
   Result := Reflect.Tiles.DistanceFromTile(Tile) < Dist;
 end;
 
-function TReflectionTiles.InPolygon(const Poly: TPointArray): Boolean;
```

Comment:
TReflectionTiles.InPolygon has a default argument of Point(-1, -1), and if the parameter is such, it will use GetGlobalTile as P.